apps/site/src/pages/[shortname]/types/entity-type/[...slug-maybe-version].page.tsx

```diff
@@ -190,53 +207,85 @@ const EntityTypePage: NextPage = () => {
-                {!isLatest && (
-                  <Link
-                    href={latestVersionUrl}
-                    onClick={(event) => {
-                      event.preventDefault();
-                      setEntityType({
-                        entityType: latestVersion,
-                        latestVersion,
-                      });
-                    }}
-                    sx={{
-                      fontWeight: 600,
-                      fontSize: "1.2rem",
-                      marginLeft: 1.5,
-                    }}
+              <Stack
+                flexDirection={{ md: "row" }}
+                spacing={{ xs: 1 }}
+                justifyContent="space-between"
+              >
+                <Stack flexDirection="row" alignItems="center">
+                  {entityTypeIsLink && (
+                    <Tooltip
+                      title="This is a 'link' entity type. It is used to link other entities together."
+                      placement="top"
+                    >
+                      <Box>
+                        <LinkIcon
+                          sx={({ palette }) => ({
+                            stroke: palette.gray[50],
+                            mr: 0.5,
+                          })}
+                        />
+                      </Box>
+                    </Tooltip>
+                  )}
+                  <Typography variant="bpHeading3" component="h1">
+                    <strong>{title}</strong> {"  "}Entity Type
+                  </Typography>
+                  <Typography
+                    variant="bpHeading3"
+                    component="span"
+                    marginLeft={1}
                   >
-                    <Typography variant="bpHeading4" color="inherit">
-                      (v{latestVersionNumber} available)
-                    </Typography>
-                  </Link>
-                )}
+                    v{entityType.metadata.recordId.version}
+                  </Typography>
+                  {!isLatest && (
+                    <Link
+                      href={latestVersionUrl}
+                      onClick={(event) => {
+                        event.preventDefault();
+                        setEntityType({
+                          entityType: latestVersion,
+                          latestVersion,
+                        });
+                      }}
+                      sx={{
+                        fontWeight: 600,
+                        fontSize: "1.2rem",
+                        marginLeft: 1.5,
+                      }}
+                    >
+                      <Typography variant="bpHeading4" color="inherit">
+                        (v{latestVersionNumber} available)
+                      </Typography>
+                    </Link>
+                  )}
+                </Stack>
+                <Box>
+                  {entityType?.schema.$id && (
+                    <Tooltip
+                      arrow
```

Comment:
```suggestion
```
@nonparibus Do we want the arrow here? Asking because we don't normally use it in other tooltips

![image](https://user-images.githubusercontent.com/37453800/225069648-20926c6b-3f38-476e-83e4-784e82af31e4.png)

![image](https://user-images.githubusercontent.com/37453800/225069573-368961e6-767c-42c3-8ae5-e68bfd379d50.png)